CHANGELOG.md

```diff
@@ -3,6 +3,106 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.2.0
```

Comment:
Do you think the new additions warrant a minor? 🤔

Reply:
Looks like a patch version to me

Reply:
If we say that "new feature" = "minor", then #12595 justifies the minor since it's not a bug fix, not an internal change, but purely something to support new use cases.

WDYT to also include this topic inside the lib release synchronization discussion? "What type of content justifies to do a minor?"

Reply:
> WDYT to also include this topic inside the lib release synchronization discussion? "What type of content justifies to do a minor?"

Makes total sense! 👍 
Maybe doing a minor release on all packages for a completely minor addition in one package is not the most optimal approach..? 🤷

Reply:
Maybe
I do agree that, as we grow, if we do a minor on every single new feature even the small one, we will end up doing minor almost every week.
Is this an issue? I don't know to be honest

We discuss feature grouping (keeping the feature un-released to do one minor per month with all the features together), would this only concern big features or would this kind of small feature also be impacted?

I don't think there is a good answer here, we must choose a release strategy and stay coherent with it.
I think that it's totally fine to only do minor every week with small new features.
I think that it's fine grouping the important features for a monthly minor release and release the small feature such as this one in patches
I think that it's fine to do minor whenever we have an important feature and just put the small feature such as this on in patches
And several other approaches are also probably viable